requirements-python.txt

```diff
@@ -26,3 +26,4 @@ pytest==4.6.4
 tcconfig==0.23.0
 selenium==3.141.0
 mysql-connector-python==8.0.19
+docker==4.2.0
```

Comment:
where do we use it?

Reply:
It installed as a dependency for `tcconfig`, but I need a newer version for my further PR #1923

Reply:
ok